sms/v1/openapi.yaml

```diff
@@ -1397,9 +1541,9 @@ components:
           type: string
           description: The identification number of the specific message part
         price:
-          type: string
+          type: number
           description: The price per message from the respective network; negative prices represent payouts in favor of a tyntec’s customer.
-          example: "0.005"
+          example: 0.005
```

Comment:
Could not test it. Is this correct?

Reply:
Nope was correct. Price is `string`

Reply:
OK, I've dropped 9aacec83086f704951750d5b2a601255daaf0373.